src/ResourceManager/Security/Commands.Security/Commands.Security.csproj

```diff
@@ -38,8 +38,17 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.Azure.Management.SecurityCenter, Version=0.10.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\Microsoft.Azure.Management.SecurityCenter.0.10.0-preview\lib\net452\Microsoft.Azure.Management.SecurityCenter.dll</HintPath>
+    <Reference Include="Microsoft.Azure.Management.SecurityCenter, Version=0.11.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
You won't need any changes here - you will need to make changes in the nectcore project